services/streammanager/kafka/kafkamanager.go

```diff
@@ -130,27 +130,31 @@ func NewProducer(destinationConfig interface{}, o Opts) (sarama.SyncProducer, er
 
 	var destConfig = Config{}
 	jsonConfig, err := json.Marshal(destinationConfig)
+	hosts := make([]string, 0)
 	if err != nil {
 		return nil, fmt.Errorf("[Kafka] Error while marshaling destination Config %+v, with Error : %w", destinationConfig, err)
 	}
 	err = json.Unmarshal(jsonConfig, &destConfig)
 	if err != nil {
 		return nil, fmt.Errorf("[Kafka] Error while unmarshalling dest config :: %w", err)
 	}
-	hostName := destConfig.HostName + ":" + destConfig.Port
+	hostNames := strings.Split(destConfig.HostName, ",")
+	for _, hostName := range hostNames {
+		hosts = append(hosts, hostName+":"+destConfig.Port)
```

Comment:
What if someone wants to provide multiple hosts with different ports? We should consider supporting a configuration similar to [boostrap.servers](https://jaceklaskowski.gitbooks.io/apache-kafka/content/kafka-properties-bootstrap-servers.html)

Reply:
I spoke with @snarkychef  on this and he  suggested it would be rare that we would have a requirement of supporting a combination of host and portNames